src/components/PageSections/Menu/_FullSiteMenu.scss

```diff
@@ -39,33 +37,46 @@
 
 .coa-ThemesNav__theme {
   margin: 0;
+  display: table;
+  position: relative;
+  text-align: left;
+  height: 56px;
+  padding: 10px 5px 10px 5px;
 }
 
 .coa-ThemesNav__link {
-  &:hover {
-    color: $coa-color-white;
-  }
+  transition: background-color .3s, box-shadow .3s;
+  display: table-cell;
+  vertical-align: middle;
+  margin: 10px;
+  border-radius: 18px;
+  padding-left: 16px;
+  padding-right: 16px;
   cursor: pointer;
   text-decoration: none;
   font-size: $coa-font-size-xsmall;
-  color: $coa-color-white;
+  font-weight: $coa-font-regular !important; // needs to override font-weight for a tags.
+  color: $coa-color-burnt-coffee-black;
   font-weight: $coa-font-medium;
-  display: block;
-  padding: $coa-spacing-medium $coa-space-level6 $coa-spacing-medium 0;
-
-  @include tablet {
-    line-height: $coa-line-height-xxsmall;
-    font-size: $coa-font-size-xxsmall;
-    height: 8rem;
+  line-height: 16px;
+  &:hover {
+    color: $coa-color-white;
+    background-color: $coa-color-primary;
+    box-shadow: 2px 5px 7px rgba(0,0,255,0.4);
+    transition: background-color .3s, box-shadow .3s;
   }
 }
 
-.coa-ThemesNav--open {
-  border-top: $coa-menu-border-width solid $coa-menu-bodercolor;
-  border-left: $coa-menu-border-width solid $coa-menu-bodercolor;
-  border-right: $coa-menu-border-width solid $coa-menu-bodercolor;
-  border-top-left-radius: 3px;
-  border-top-right-radius: 3px;
+.coa-ThemesNav__link.es {
+  font-size: 13px;
+  line-height: 13px;
+  &:hover {
+    color: $coa-color-white;
+    background-color: $coa-color-primary;
+    box-shadow: 2px 5px 7px rgba(0,0,255,0.4);
+    transition: background-color .3s, box-shadow .3s;
+  }
+  transition: background-color .3s, box-shadow .3s;
 }
```

Comment:
Feels like a lot is copy/pasted between .coa-ThemesNav__link and .coa-ThemesNav__link.es. Is all that necessary, or just the font size changes?

Reply:
Good eyes! Yep.